networks/suzuka/indexer/src/main.rs

```diff
@@ -124,8 +149,18 @@ default_sleep_time_between_request: {}
 	// })?;
 	write!(output_file, "{}", indexer_config_content)?;
 
-	let indexer_config =
+	let mut indexer_config =
 		server_framework::load::<IndexerGrpcProcessorConfig>(&output_file.path().to_path_buf())?;
+
+	// Use to print the generated config, to have an example when activating a new processor.
+	// indexer_config.processor_config = ProcessorConfig::TokenV2Processor(TokenV2ProcessorConfig {
+	// 	query_retries: 5,
+	// 	query_retry_delay_ms: 100,
+	// });
+
+	// let yaml = serde_yaml::to_string(&indexer_config)?;
+	// println!("{yaml}",);
+
```

Comment:
Same here, let's try to avoid this habit of leaving commented out code in main.

Reply:
I leave this code because I still need it from time to time when debugging some issue with the config file. I didn't know where to put it. When all basic issue of the indexer will be solved, I won't need it.

Reply:
Ok